### PR TITLE
Add novas package

### PR DESCRIPTION
**Pull request by treloar**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by treloar** [comment]
Created the PR/happy to be a maintainer.

**Comment by treloar** [comment]
@conda-forge-linter This package should not be licensed, refer to "License and Citation" section of https://pypi.org/project/novas/.

To quote there:
"This software was produced by the United States Naval Observatory at the expense of United States taxpayers, and is therefore not suseptible to copyright, because a copyright would place taxpayer property under private ownership. Since it is not copyrighted, it cannot be licensed; it is simply free."

**Comment by treloar** [comment]
Howdy @conda-forge/staged-recipes , pretty sure this PR is ready for a review.

Thanks :)

**Comment by treloar** [comment]
@conda-forge-admin, please ping team

Howdy again, not sure if this is spam. First time contributor and I believe my earlier ping to @conde-forge/staged-recipes will not be doing anything.

Cheers,
treloar

File: recipes/novas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+test:
+  imports:
+    - novas
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://pypi.org/project/novas/3.1.1.5/
```
Comment (chrisburr) [inline recipes/novas/meta.yaml:35]:
```suggestion
  home: https://pypi.org/project/novas/
```

File: recipes/novas/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{% set name = "novas" %}
+{% set version = "3.1.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/42/95/a05bc35cb119925e10f9faa8a2bd17020b0a585744a38921a709acdd9a14/{{ name }}-{{ version }}.tar.gz
```
Comment (chrisburr) [inline recipes/novas/meta.yaml:9]: This should be the predictable URL (see `recipes/example/meta.yaml` for how to construct it)

File: recipes/novas/LICENSE [hunk 3]
```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
```
Comment (chrisburr) [inline recipes/novas/LICENSE:1]: This looks like it should be public domain with the license file being:
https://github.com/brandon-rhodes/python-novas/blob/2e690e925e53ed585f57b3105cf92ea70954b60e/README-PyPI#L210-L226

File: recipes/novas/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    - novas
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://pypi.org/project/novas/3.1.1.5/
+  summary: 'NOVAS is an integrated package of functions for computing various commonly needed quantities in positional astronomy.'
+  license: MIT
```
Comment (chrisburr) [inline recipes/novas/meta.yaml:37]: The text looks close to `NIST-PD` I would go with
```suggestion
  license: LicenseRef-USNO-PD
  license_family: PUBLIC-DOMAIN
```

File: recipes/novas/meta.yaml (hunk 1 above)
Comment (treloar) [reply recipes/novas/meta.yaml]: Good idea, done.

File: recipes/novas/meta.yaml (hunk 2 above)
Comment (treloar) [reply recipes/novas/meta.yaml]: Done, copied from `recipes/example/meta.yml`

File: recipes/novas/LICENSE (hunk 3 above)
Comment (treloar) [reply recipes/novas/LICENSE]: You are very correct, done.

File: recipes/novas/meta.yaml (hunk 4 above)
Comment (treloar) [reply recipes/novas/meta.yaml]: Yes absolutely correct, done.

**Comment by treloar** [comment]
@chrisburr  Thank you for the review. I've gone through and implemented all your suggestions.

**Pull request merged**